client/blocks/calendar-popover/README.md

```diff
@@ -11,12 +11,12 @@ import { Button } from '@automattic/components';
 import CalendarPopover from 'blocks/calendar-popover';
 
 const toggle = () => this.setState( { show: ! this.state.show } );
-let buttonRef;
+const buttonRef = React.createRef();
```

Comment:
I'm assuming this code is in a `React.Component` context (because it has a `render()` function), so those `const` here are a bit odd. If we disregard this, usage seems good 👍